projects/sky-autonumeric/src/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -230,7 +228,7 @@ export class SkyAutonumericDirective
   }
 
   /* istanbul ignore next */
-  private onChange = (_: number) => {};
+  private onChange = (_?: number) => {};
```

Comment:
Should this be `number | undefined` so that the parameter is still required? It looks like we specify the parameter everywhere we call this internally, so it might be nice to keep that pattern and require passing `undefined` explicitly instead of leaving out the parameter altogether.